src/backend/distributed/commands/function.c

```diff
@@ -537,27 +542,33 @@ GetFunctionDDLCommand(const RegProcedure funcOid)
 	StringInfo ddlCommand = makeStringInfo();
 
 	OverrideSearchPath *overridePath = NULL;
-	Datum sqlTextDatum = 0;
 	char *createFunctionSQL = NULL;
 	char *alterFunctionOwnerSQL = NULL;
 
-	/*
-	 * Set search_path to NIL so that all objects outside of pg_catalog will be
-	 * schema-prefixed. pg_catalog will be added automatically when we call
-	 * PushOverrideSearchPath(), since we set addCatalog to true;
-	 */
-	overridePath = GetOverrideSearchPath(CurrentMemoryContext);
-	overridePath->schemas = NIL;
-	overridePath->addCatalog = true;
-	PushOverrideSearchPath(overridePath);
+	createFunctionSQL = GetAggregateDDLCommand(funcOid);
```

Comment:
As noted below;

This function will be more readable if we use `get_func_prokind` to get the function type and differentiate between calling `GetAggregateDDLCommand` and the body below. Instead of using the `NULL` output here to fallback to the other behaviour.